crates/lgn-graphics-renderer/src/core/render_graph/render_graph.rs

```diff
@@ -399,17 +399,17 @@ pub struct ResourceData {
     pub load_state: RenderGraphLoadState,
 }
 
-pub(crate) type RenderGraphExecuteFn =
+pub type RenderGraphExecuteFn =
     dyn Fn(&RenderGraphContext, &mut RenderGraphExecuteContext<'_, '_>, &mut CommandBuffer);
 
-pub(crate) struct RGNode {
```

Comment:
This comment applies to most of the following changes. 
If something needs to be exposed to the crate, it should be in the mod.rs. Putting pub(crate) everywhere is verbose and not really needed. Need to discuss that before we do a full cleanup pass.

Reply:
OK